TFVC_README.md

```diff
@@ -0,0 +1,198 @@
+assigned to you (or returned from your custom query) will be displayed for you to choose. Choosing one adds it to your check-in comment.
+* `Check In` – Use this option to check in your changes.
+* `Undo All` - If there are file modifications, this command will prompt for confirmation and then undo all of the changes.
+* `Show TFVC Output` – Use this option to display the `TFVC Output` window which shows the TFVC commands run during this session.
+* `Switch SCM Provider... >` – This option is provided by VS Code and allows you to change between the available source control
+providers (e.g, Git, TFVC, etc.).
+
+#### Resolving Conflicts
+When conflicts need to be resolved, you can use these commands to do so. There are inline icons which also provide this functionality.
+
+![TFVC Resolve Conflicts](assets/tfvc-resolve-conflicts.png)
+
+* `Open Diff` – Opens the diff view between the modified file and the versioned file.
+* `Open File` – Opens the modified file in the editor.
+* `Resolve: Take Theirs` – Resolves the conflict as `Take Theirs`.
+* `Resolve: Keep Yours` – Resolves the conflict as `Keep Yours`.
+* `Undo` – Reverts the changes made to the file.
+
+***Note:*** Resolving conflicts between branches is not supported in the extension. At this time, these conflicts are best addressed
+in an IDE or at the command line.
+
+### File Explorer
+Access to renaming a file is provided on Visual Studio Code's File Explorer menu.
+
+![TFVC Rename](assets/tfvc-rename.png)
+
+* `Rename (TFVC)` – To properly rename a file in TFVC using the `Rename` command (instead of `Delete` and `Add`), use this option on
+Visual Studio Code's File Explorer. You will prompted for a new file name.
+
+## Additional Information
+### Specifying a TFS Proxy
+To configure a TFS proxy server, you must provide the URL to that server in the Visual Studio Code Settings (**File > Preferences > Settings**):
+  ```javascript
+    "tfvc.proxy": "http://servername:9999/proxy"
+  ```
+
+## Frequently Asked Questions
+### *Is it required that I have a TFVC workspace already on my local machine to use the TFVC support?*
+With release 1.116.0, yes, it is a requirement that you have an existing workspace on your local machine.
+
+### *Can I use the Team Explorer Everywhere Command Line Client (TEE CLC) to provide the TFVC functionality on Windows?*
+Yes. If you use Eclipse or one of JetBrain's IDEs (e.g, Android Studio, IntelliJ) on Windows, then you will want to use the TEE CLC to provide
+the TFVC support.
+
+### *Which instructions do I follow to set up TFVC functionality on the macOS?*
+You will follow the same instructions for setting up the TEE CLC as is shown on Ubuntu in [this video](https://youtu.be/VPNaEIVZfr0).
+
+### *I have workspaces created with Visual Studio. Can I use the TEE CLC to work with them?*
+This should be possible. However, you will need to make the TEE CLC aware of those workspaces by running the `tf workspaces` command.
```

Comment:
I think you have to specify the collection with the tf workspaces command to get it to recognize the ones that are not cached.